src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/TrumpetSchwarzschild.hpp

```diff
@@ -32,14 +32,48 @@ namespace gr::Solutions {
  *
  * \details
  * This solution is a trumpet Schwarzschild black hole in the isotropic
- * coordinates. It is time-independent puncture solution in 1+log slicing.
+ * coordinates. It is a time-independent puncture solution in 1+log slicing.
  * The solution cannot be written down analytically in Schwarzschild
  * coordinates or isotropic coordinates. Refer to \cite Hannam2008
  * for equations and details.
  *
+ * We first set up a source grid in isotropic radial coordinate r, on which
+ * we compute the lapse and Schwarzschild radial coordinate R. The lapse
+ * is computed by solving eq. (54)-(56) using the toms748 algorithm, where
```

Comment:
Would it be easy to include the equations directly in the dox? You should be able to download the tex file from arxiv to make it easier. Then at least this class is fairly standalone :)